### ISSUE TITLE
Question: Tensorflow 2.1 supported by R Tensorflow Package?

### ISSUE THREAD

**Issue by KlausPuchner** [rstudio/tensorflow#415]
Hi Everyone,

I am setting up a dockerized RStudio Server environment. Each container gets RStudio Server and a particular gpu-powered TF version (TF1.14, TF1.15.2, TF2.0.1, TF2.1.0) as backend for deep learning tasks. This implies that I use the tensorflow packages that are shipped with the respective TF docker images (which should not be a problem since those are the same tensorflow pip packages).

As I already mentioned, I used the respective official Tensorflow Containers. All TF versions work fine when using RStudio server with the R Packages Tensorflow 2.0.0 and Keras 2.2.5.0 **except** TF2.1.

I also built a second separate environment (same TF Versions and Base Images) for Jupyterlab. All TF versions work just fine and also take advantage of the gpu. So the TF2.1.0 Docker base image itself is working fine.

So I was wondering: Which TF Versions are officially supported by the R Tensorflow Package? Or might the R Keras Package be the reason for TF2.1 not working in RStudio?

Please advise, thanks in advance. :)

**Comment by dfalbel** [comment]
Yes, TensorFlow for R supports TF 2.1 and it's currently in our test matrix. For keras, we have a known bug regarding generator R functions (https://github.com/rstudio/keras/issues/986).

**Comment by KlausPuchner** [comment]
Thank you very much for the info. :)
